core/src/main/java/org/infinispan/remoting/inboundhandler/GlobalInboundInvocationHandler.java

```diff
@@ -101,10 +101,14 @@ public void handleFromRemoteSite(String origin, XSiteRequest<?> command, Reply r
    }
 
    private void handleCacheRpcCommand(Address origin, CacheRpcCommand command, Reply reply, DeliverOrder mode) {
+      ByteString cacheName = command.getCacheName();
+      if (cacheName == null) {
```

Comment:
The testsuite passes without this change. I'm not expecting to have a null cache name.